### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by andresporras3423**
- The website was built with JavaScript, Webpack, and Open Weather API
- You can search for weather condition of any city
- The index page has a form to search a city and select unit format for temperature information
- The output for the search will give a list of related data to the city's weather: temperature, wind speed, pressure, humidity.
- Output shows an icon to represent the cloud condition of the city, it also updates the background image of the page.
- The current project uses async methods, promises, and ECMA Script 6

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @andresporras3423  👋

**You did quite well on this. Separating dom-manipulation from business logic is a good one. 👌**

However, let's make some fixes.

- [ ] IMPORTANT - Enable eslint GitHub actions in your project and ensure your project is properly linted.

- [ ]  After toggling temperature the temperature value doesn't change. Please fix it.

- [ ] Refer to other comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/page-load.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    this.divCenter.appendChild(this.hTitle);
+    this.divCenter.appendChild(this.pExplain);
+    this.divCenter.appendChild(this.divSearch);
+    this.divCenter.appendChild(this.divList);
+    this.divContent.appendChild(this.divLeft);
+    this.divContent.appendChild(this.divCenter);
+    this.divContent.appendChild(this.divRight);
+    this.loadDivShowResult();
+    this.loadDivSearchCity();
+    this.buttonSearch.onclick = function buttonSearchClick() { that.getWeather(); };
```
Comment (mikenath223) [inline src/page-load.js]: - [ ] Please replace this with es6 arrow function suntax.

**Comment by andresporras3423** [comment]
Thank you TSE for the feedback, I have fixed the three highlighted issues:
- Linters are working
- The information includes a toggle to switch between Celsius and Fahrenheit
- using arrow functions for onclick events

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**